remote-config.gemspec

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+lib = File.expand_path("lib", __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "remote_config/version"
+
+Gem::Specification.new do |spec|
+  spec.name        = "remote_config"
+  spec.version     = RemoteConfig::VERSION
+  spec.authors     = ["Montgomery Anderson"]
+  spec.email       = "montgomery.c.anderson@gmail.com"
```

Comment:
What are the authors and email using for?

Reply:
Umm, I think it's just required to be listed on https://rubygems.org and is auto-generated. It's in case someone wants to contact the authors for whatever reason.